app/js/checkAnswers.ts

```diff
@@ -140,25 +127,22 @@ function showResults(pageLeft) {
     clearInterval(interval)
     const userAnswers = getUserAnswers()
     checkAnswers(userAnswers).then(function (result) {
-        let percentResult
-        let answered
         if (result.score || result.score === 0) {
             if (pageLeft) {
-                document.querySelector<HTMLElement>('.userMessage').innerHTML = `<h1>Test cancelled!</h1>
+                document.querySelector<HTMLElement>('.greetings').innerHTML = `<p>Test cancelled!</p>`
+                document.querySelector<HTMLElement>('.email_for_results').innerHTML = `
                 <p>This test has ended because you clicked away from the page</p>
-                <p>Please contact the office to discuss further</p>`
+                <p>Please contact the office at <a href="mailto:hello@mayden.academy">hello@mayden.academy</a> to discuss further</p>`
                 result.score = 0
             } else {
-                document.querySelector<HTMLElement>('.userMessage').innerHTML = `<h1>Thank You!</h1>
-                <p>You have completed the test!</p>
-                <p>Please contact the office if you would like to find out your results</p>`
+                document.querySelector<HTMLElement>('.greetings').innerHTML = `<p>Thank You!</p>`
+                document.querySelector<HTMLElement>('.email_for_results').innerHTML = `
+                <p id="completedMessage">You have completed the test!</p>
+                <p>Please contact the office at <a href="mailto:hello@mayden.academy">hello@mayden.academy</a> if you would like to find out your results</p>`
```

Comment:
Please switch your backticks for normal single quotes to match the rest of the file. We only need to use the backticks when making use of template literals!